lambdas/services/create_document_reference_service.py

```diff
@@ -30,7 +30,8 @@
 
 class CreateDocumentReferenceService:
     def __init__(self):
-        self.s3_service = S3Service()
+        create_document_aws_role_arn = os.getenv("PRE_SIGN_ASSUME_ROLE")
+        self.s3_service = S3Service(custom_aws_role=create_document_aws_role_arn)
```

Comment:
Just an idea, would it be good to change this `custom_aws_role` as a new argument of presign url method e.g. create_upload_presigned_url, rather than passing it during service initialisation?

The reasons that I suggest this:
 - If another lambda wants to use presign url, the method signature for e.g `create_upload_presigned_url` would clearly indicate that a `custom_aws_role` is necessary for that method to work
 - S3Service can be initialised in the same way `S3Service()` throughout the codebase, which would be better as that service is singleton